### PR TITLE
Enable Infrasim-init command:

### PR DESCRIPTION
**Pull request by sharkconi**
1. create related directory
2. render config file under /etc/infrasim
3. mv repo/{data,conf} to repo/data/{conf,script}

Signed-off-by: sharkconi <sharkconi@gmail.com>

### CODE REVIEW

**Comment by sharkconi** [comment]
@fub2 @xiar @turtle-fly @PaynePei @JuneZhou19

**Comment by turtle-fly** [comment]
:+1:  
Now default configuration is recorded in configuration file, instead of in vm.py, etc.

**Comment by fub2** [comment]
Practicing and started from a vagrant box, official xenial, several issue encountered when running command "sudo infrasim-mail start"

1. /usr/local/bin/ipmi_sim: error while loading shared libraries: libOpenIPMIposix.so.0: cannot open shared object file: No such file or directory
--> sudo apt-get install ipmitool  seems resolved it
2. /usr/local/bin/ipmi_sim: symbol lookup error: /usr/local/bin/ipmi_sim: undefined symbol: persist_enable  
--> No idea how to resolve it

File: bin/infrasim-init [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+def create_infrasim_directories():
+    if os.path.exists(os.environ['HOME'] + '/.infrasim') is False:
+        os.mkdir(os.environ['HOME'] + "/.infrasim")
+
+    if os.path.exists("/etc/infrasim") is False:
+        os.mkdir("/etc/infrasim")
+
+def create_infrasim_conf():
+    nics_list = netifaces.interfaces()
+    eth_nic = filter(lambda x: 'e' in x,nics_list)[0]
```
Comment (PaynePei) [inline bin/infrasim-init]: how to handle with the case that filter(lambda x: 'e' in x,nics_list) returns []?

File: bin/infrasim-init [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+INFRASIM_TEMP_CONF = "/usr/local/etc/infrasim/conf/infrasim.conf"
+INFRASIM_CONF = "/etc/infrasim/infrasim.conf"
+VBMC_TEMP_CONF = "/usr/local/etc/infrasim/conf/vbmc.conf"
+VBMC_CONF = "/etc/infrasim/vbmc.conf"
+
+mac_base = "00:1e:67:"
+def create_mac_address():
+    macs = []
+    for i in range(0, 9):
+        macs.append(''.join(random.SystemRandom().choice("abcdef" + string.digits) for _ in range(2)))
```
Comment (PaynePei) [inline bin/infrasim-init]: :+1:

**Pull request merged**